lib/verify/metadata/generator.rb

```diff
@@ -26,21 +26,11 @@ def self.run!(arguments)
         valid_until = DateTime.now + options.valid_until
         xsd_validator = XsdValidator.new(METADATA_XSD)
         options.environments.each do |environment|
-          if options.is_connector
-            hub_entity_descriptor = hub_metadata_provider.provide(environment, valid_until)
```

Comment:
It looks like we're no longer using the `HubMetadataProvider.provide` and `IdpMetadataProvider.provide` functions with the `valid_until` parameter set anywhere in the codebase. Can we tidy up these functions by removing the `valid_until` parameters and any other relevant code?